Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -944,8 +944,8 @@ def _update(self):
 
         if distances is not None:
             # Convert to flat upper triangular distances
-            i, j = numpy.triu_indices(distances.X.shape[0], k=1)
-            distances = distances.X[i, j]
+            i, j = numpy.triu_indices(distances.shape[0], k=1)
+            distances = numpy.asarray(distances[i, j])
```

Comment:
Sure, but that's because `D[i, j]` is no longer a symmetric matrix; it's a 1D array constucted anew from `i` and `j` indices. It's the same with
```py
>>> A = np.arange(D.size).reshape(D.shape)
>>> A[i, j]
array([ 1,  2,  3,  6,  7, 11])
```
There is no problem, right?